### ISSUE TITLE
Generic image tags

### ISSUE THREAD

**Issue by Starefossen** [eclipse/jetty.docker#69]
Thanks for making this image 💯  I am wondering if it is possible to get more generic image tags like this:

* `jetty:11-jdk11-slim`
* `jetty:10-jdk11-slim`
* etc.

Some exists already, but quit a few seams to be missing.

**Comment by joakime** [comment]
Looks like all of the following shorthand tags are missing (unlike their 9.x cousins)

```
10-<jdk>-<type>
10.<major_ver>-<jdk>-<type>
11-<jdk>-<type>
11.<major_ver>-<jdk>-<type>
```

**Comment by steinard** [comment]
And an image for jetty:11-jdk17-slim

**Comment by lachlan-roberts** [comment]
The fix for this from PR #70 has been merged, the new tags will be available after the next release for Jetty which should be in the next few days.
